### PR TITLE
[pt_BR] Machine translation of missing keys

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

**Comment by caouecs** [comment]
@HabiboIV can you help us and review this automatic pull request, please ? thanks by advance

File: locales/pt_BR/validation-inline.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
     'alpha_dash'           => 'Este campo pode conter apenas letras, números, traços e sublinhados.',
     'alpha_num'            => 'Este campo pode conter apenas letras e números',
     'array'                => 'Este campo deve ser uma matriz.',
-    'attached'             => 'This field is already attached.',
+    'attached'             => 'Este campo já está ligado.',
```
Comment (HabiboIV) [inline locales/pt_BR/validation-inline.php:23]: "Este campo já está anexado." sounds better.

File: locales/pt_BR/validation.php [hunk 2]
```diff
@@ -20,7 +20,7 @@
     'alpha_dash'           => 'O campo :attribute deve conter apenas letras, números e traços.',
     'alpha_num'            => 'O campo :attribute deve conter apenas letras e números .',
     'array'                => 'O campo :attribute deve conter um array.',
-    'attached'             => 'This :attribute is already attached.',
+    'attached'             => 'Este :attribute já está ligado.',
```
Comment (HabiboIV) [inline locales/pt_BR/validation.php:23]: "Este :attribute já está anexado." sounds better.

**Review by HabiboIV — CHANGES_REQUESTED** [review]

**Comment by caouecs** [comment]
Ok, I will fix this evening

and for the json file ?

File: locales/pt_BR/pt_BR.json [hunk 3]
```diff
@@ -139,480 +139,480 @@
...
     "Guernsey": "Guernsey",
-    "Guinea": "Guinea",
-    "Guinea-Bissau": "Guinea-Bissau",
-    "Guyana": "Guyana",
-    "Haiti": "Haiti",
-    "Heard Island & Mcdonald Islands": "Heard Island and McDonald Islands",
+    "Guinea": "Guine",
+    "Guinea-Bissau": "Guine",
+    "Guyana": "Guianense",
+    "Haiti": "Feira",
```
Comment (HabiboIV) [inline locales/pt_BR/pt_BR.json:229]: "Haiti"

File: locales/pt_BR/pt_BR.json [hunk 4]
```diff
@@ -139,480 +139,480 @@
...
+    "Guatemala": "Guatemalteco",
     "Guernsey": "Guernsey",
-    "Guinea": "Guinea",
-    "Guinea-Bissau": "Guinea-Bissau",
-    "Guyana": "Guyana",
-    "Haiti": "Haiti",
-    "Heard Island & Mcdonald Islands": "Heard Island and McDonald Islands",
+    "Guinea": "Guine",
+    "Guinea-Bissau": "Guine",
+    "Guyana": "Guianense",
```
Comment (HabiboIV) [inline locales/pt_BR/pt_BR.json:228]: "Guiana"

File: locales/pt_BR/pt_BR.json [hunk 5]
```diff
@@ -139,480 +139,480 @@
...
-    "Guatemala": "Guatemala",
+    "Guatemala": "Guatemalteco",
     "Guernsey": "Guernsey",
-    "Guinea": "Guinea",
-    "Guinea-Bissau": "Guinea-Bissau",
-    "Guyana": "Guyana",
-    "Haiti": "Haiti",
-    "Heard Island & Mcdonald Islands": "Heard Island and McDonald Islands",
+    "Guinea": "Guine",
+    "Guinea-Bissau": "Guine",
```
Comment (HabiboIV) [inline locales/pt_BR/pt_BR.json:227]: "Guiné-Bissau"

File: locales/pt_BR/pt_BR.json [hunk 6]
```diff
@@ -139,480 +139,480 @@
...
     "Guam": "Guam",
-    "Guatemala": "Guatemala",
+    "Guatemala": "Guatemalteco",
     "Guernsey": "Guernsey",
-    "Guinea": "Guinea",
-    "Guinea-Bissau": "Guinea-Bissau",
-    "Guyana": "Guyana",
-    "Haiti": "Haiti",
-    "Heard Island & Mcdonald Islands": "Heard Island and McDonald Islands",
+    "Guinea": "Guine",
```
Comment (HabiboIV) [inline locales/pt_BR/pt_BR.json:226]: "Guiné"

File: locales/pt_BR/pt_BR.json [hunk 7]
```diff
@@ -139,480 +139,480 @@
...
     "Holy See (Vatican City State)": "Vatican City",
     "Honduras": "Honduras",
-    "Hong Kong": "Hong Kong",
-    "Hungary": "Hungary",
-    "I agree to the :terms_of_service and :privacy_policy": "I agree to the :terms_of_service and :privacy_policy",
-    "Iceland": "Iceland",
+    "Hong Kong": "reformado",
+    "Hungary": "Hungria",
+    "I agree to the :terms_of_service and :privacy_policy": "Concordo com o :terms_of_service e com o :privacy_policy",
+    "Iceland": "Islandês",
```
Comment (HabiboIV) [inline locales/pt_BR/pt_BR.json:239]: "Islândia"

File: locales/pt_BR/pt_BR.json [hunk 8]
```diff
@@ -139,480 +139,480 @@
...
-    "Hold Up!": "Hold Up!",
+    "Hide Content": "Esconder O Conteúdo",
+    "Hold Up!": "Espera!",
     "Holy See (Vatican City State)": "Vatican City",
     "Honduras": "Honduras",
-    "Hong Kong": "Hong Kong",
-    "Hungary": "Hungary",
-    "I agree to the :terms_of_service and :privacy_policy": "I agree to the :terms_of_service and :privacy_policy",
-    "Iceland": "Iceland",
+    "Hong Kong": "reformado",
```
Comment (HabiboIV) [inline locales/pt_BR/pt_BR.json:236]: "Hong Kong"

File: locales/pt_BR/pt_BR.json [hunk 9]
```diff
@@ -139,480 +139,480 @@
...
-    "If you do not have an account, you may create one by clicking the button below. After creating an account, you may click the invitation acceptance button in this email to accept the team invitation:": "If you do not have an account, you may create one by clicking the button below. After creating an account, you may click the invitation acceptance button in this email to accept the team invitation:",
+    "If you do not have an account, you may create one by clicking the button below. After creating an account, you may click the invitation acceptance button in this email to accept the team invitation:": "Se você não tem uma conta, você pode criar uma clicando no botão abaixo. Depois de criar uma conta, você pode clicar no botão aceitação do convite neste e-mail para aceitar o convite da equipe:",
     "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Se você estiver tendo problemas para clicar no botão \":actionText\", copie e cole a URL abaixo\nem seu navegador:",
-    "Increase": "Increase",
-    "India": "India",
-    "Indonesia": "Indonesia",
+    "Increase": "Aumentar",
+    "India": "Indio",
```
Comment (HabiboIV) [inline locales/pt_BR/pt_BR.json:251]: "India"

File: locales/pt_BR/pt_BR.json [hunk 10]
```diff
@@ -139,480 +139,480 @@
...
+    "Indonesia": "Indonesio",
```
Comment (HabiboIV) [inline locales/pt_BR/pt_BR.json:252]: "Indonesia"

File: locales/pt_BR/pt_BR.json [hunk 11]
```diff
@@ -139,480 +139,480 @@
...
     "Invalid signature.": "Assinatura inválida.",
-    "Iran, Islamic Republic Of": "Iran",
-    "Iraq": "Iraq",
-    "Ireland": "Ireland",
-    "Isle Of Man": "Isle of Man",
+    "Iran, Islamic Republic Of": "Iraniano",
```
Comment (HabiboIV) [inline locales/pt_BR/pt_BR.json:254]: "Irã"

**Review by HabiboIV — CHANGES_REQUESTED** [review]
more review is needed. from line 255 onwards

**Comment by caouecs** [comment]
Obrigado

Can you do a new pull request starting of this pull request, with your fixes ?

**Comment by HabiboIV** [comment]
sure. I'll do just that. you will have it in 24h.

**Comment by HabiboIV** [comment]
hello there. i am new to this. 
i have done the corrections. however, i do not have permission to push my changes to "HEAD:patch/2021-04-17/pt_BR"

**Comment by HabiboIV** [comment]
@andrey-helldar, I have finished reviewing the entirety of json file. should I make a pull to your repository?

**Comment by HabiboIV** [comment]
@andrey-helldar , I have done a pull request with all my changes to your repository.

**Comment by andrey-helldar** [comment]
@HabiboIV, thank you. I accepted it.

@caouecs, the changes are in this PR.

@HabiboIV, now everything is fine?

**Comment by HabiboIV** [comment]
@andrey-helldar , yes, everything is fine now! thank you!

**Comment by caouecs** [comment]
Obrigado

**Comment by andrey-helldar** [comment]
@HabiboIV, no you!)

cover2

**Pull request merged**